catkin_ws/src/propulsion/src/drytest.py

```diff
@@ -94,9 +102,12 @@ def re_arm():
         forwards_test(7)
```

Comment:
Can't you fully remove this first option (if) and the forwards_test function? If you want to do 1-8 in sequence, you can just press 1-2-3-4-5-6-7-8 on your keyboard. No need to keep pressing next for forward_test

Reply:
Yes, but I think this is nice to have for communication between the people on the fans and us who are running the tests. I thought it may be nice that we have it so we can tell them "Running 1, running 2, repeating 2, running 5, etc." without having the restriction of going in order. If we get rid of the if, it will just run itself without us communicating "okay 7 is running, 5 is running". Sure, you can add some print statements but a lot of the time they will be like "wait run that back again", then you would have to wait until your sequence is done until you can.

Reply:
Also, i am not using forwards test, I created a new function optimized_test() so maybe u r looking at wrong commit

Reply:
nw, i'm just suggesting changes. If you think there is a better way, feel free to do it